pkg/inject/report.go

```diff
@@ -87,3 +83,35 @@ func checkUDPPorts(t *v1.PodSpec) bool {
 	}
 	return false
 }
+
+func (r *Report) disableByAnnotation(conf *ResourceConfig) bool {
+	// truth table of the effects of the inject annotation:
+	//
+	// namespace | pod      | inject?  | return
+	// --------- | -------- | -------- | ------
+	// enabled   | enabled  | yes      | false
+	// enabled   | ""       | yes      | false
+	// enabled   | disabled | no       | true
+	// disabled  | enabled  | yes      | false
+	// ""        | enabled  | yes      | false
+	// disabled  | disabled | no       | true
+	// ""        | disabled | no       | true
+	// disabled  | ""       | no       | true
+	// ""        | ""       | no       | true
+	//
+	// for CLI, only the 'disabled' annotation is taken into consideration
+	// for its opt-out effect
```

Comment:
Thanks for clarifying all this. To be even more extra super clear, I'd only prefix that table spelling out that it only applies to webhook injection.